{"historyId": "c08addc26bd2688dfb4a2b1b4eb995ad", "fullName": "test_case.test_login_py.TestLogin_send#test_login_error[case0]", "status": "passed", "uuid": "c0e53a3d-d952-4788-808f-d49d6b36d243", "stop": 1565768033056, "parameters": [{"value": "{'phone': '18684720552', 'expected': '此账号没有经过授权，请联系管理员!', 'password': 'python'}", "name": "case"}], "name": "test_login_error[case0]", "labels": [{"value": "login", "name": "tag"}, {"value": "TestLogin_send", "name": "subSuite"}, {"value": "test_case", "name": "parentSuite"}, {"value": "test_login_py", "name": "suite"}, {"value": "PC-201807091748", "name": "host"}, {"value": "8944-MainThread", "name": "thread"}, {"value": "pytest", "name": "framework"}, {"value": "cpython3", "name": "language"}, {"value": "test_case.test_login_py", "name": "package"}], "start": 1565768029902}